helm/resource_release.go

```diff
@@ -1186,6 +1218,35 @@ func getValues(d resourceGetter) (map[string]interface{}, error) {
 	return base, logValues(base, d)
 }
 
+func getListValue(base, set map[string]interface{}) error {
+	name := set["name"].(string)
+	listValue := set["value"].([]interface{}) // this is going to be a list
+	valueType := set["type"].(string)
+
+	var listString string
+	for i, val := range listValue {
+		listString += val.(string)
+		if i != len(listValue)-1 {
+			listString += ","
+		}
+	}
```

Comment:
This logic can be replaced by [`strings.Join()`](https://pkg.go.dev/strings#Join).

Reply:
looks like we can't use `strings.Join()` since we can't convert `[]interface{}` to `[]string`. I was able to clean it up a bit more with: 
```
listString := make([]string, len(listValue))
	for i, s := range listValue {
		listString[i] = s.(string)
	}
```